### ISSUE TITLE
Undesirable newlines in expected/got output

### ISSUE THREAD

**Issue by pelevesque** [lucs/Test-Selector#1]
When outputting an error, tsel sometimes adds undesirable newlines in the expected/got output.

Here is an example result with tsel:

```
# expected: Buf.new(77,84,114,107,0,0,0,16,100,145,60,100,0,129,60,10,100,145,60,40,0,255,242,0)
#      got: Buf.new(77,84,114,
107,0,0,0,16,100,145,60,100,0,129,60,10,0,145,60,40,0,255,242,0)
```

Here is the same example run with Raku's normal testing environment:

```
# expected: Buf.new(77,84,114,107,0,0,0,16,100,145,60,100,0,129,60,10,100,145,60,40,0,255,242,0)
#      got: Buf.new(77,84,114,107,0,0,0,16,100,145,60,100,0,129,60,10,0,145,60,40,0,255,242,0)
```

**Comment by lucs** [comment]
Indeed it does. After trying out similar examples, it appears that a spurious line break is introduced (here, after "114,") when the number of characters of the lines (the "# expected..." and the "#      got..." lines, and including the newline between them) reaches 127. Weird. Will have to look into it.

**Comment by lucs** [comment]
Fixed in 0.1.5. I'd made a mistake when ~~cargo-culting~~ copying some code.
